### PR TITLE
Feature/localization consolidation

### PR DESCRIPTION
**Pull request by jan-schutte**

### CODE REVIEW

File: src/Nox.Localization.SqlServer/Nox.Localization.SqlServer.csproj
Comment (rochar) [inline src/Nox.Localization.SqlServer/Nox.Localization.SqlServer.csproj:1]: So can we also join all this projects in Nox.Localization?

File: src/Nox.Localization/DbContext/NoxLocalizationDbContext.cs [hunk 1]
```diff
@@ -1,69 +1,66 @@
...
-            _dbProvider.ConfigureDbContext(optionsBuilder, appName, dbServer);
+            _databaseProvider!.ConfigureDbContext(optionsBuilder, appName, dbServer);
         }
-
-        //optionsBuilder.MigrationsAssembly("ImportExportLocalization");
-
+        
     }
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
```
Comment (rochar) [inline src/Nox.Localization/DbContext/NoxLocalizationDbContext.cs:43]: do you want to delete this method, everything is commented, and we are not going by this path?

File: src/Nox.Localization/Nox.Localization.csproj [hunk 2]
```diff
@@ -20,6 +20,10 @@
         <PackageIcon>nox.png</PackageIcon>
     </PropertyGroup>
 
+    <ItemGroup>
```
Comment (rochar) [inline src/Nox.Localization/Nox.Localization.csproj:23]: I did not verify, but this dependency seems that it should not be needed? can you confirm?

File: src/Nox.Localization/Nox.Localization.csproj [hunk 3]
```diff
@@ -30,13 +34,15 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Nox.Lib\Nox.Lib.csproj" />
+        <ProjectReference Include="..\Nox.Abstractions\Nox.Abstractions.csproj" />
+        <ProjectReference Include="..\Nox.EntityFramework.Sqlite\Nox.EntityFramework.Sqlite.csproj" />
```
Comment (rochar) [inline src/Nox.Localization/Nox.Localization.csproj:38]: yup... so lets add all Localization projects to Nox.Localization we are ok in bringing all dependencies, so....

File: src/SampleWebApp/Controllers/WeatherForecastController.cs
Comment (rochar) [inline src/SampleWebApp/Controllers/WeatherForecastController.cs:1]: I guess this was a merge conflict, you can delete [WeatherForecastController.cs]

File: tests/Nox.Localization.SqlServer.Tests/Nox.Localization.SqlServer.Tests.csproj
Comment (rochar) [inline tests/Nox.Localization.SqlServer.Tests/Nox.Localization.SqlServer.Tests.csproj:1]: so following the same idea, can we have a single Localization.Tests project?

**Review by rochar — CHANGES_REQUESTED** [review]
looks good, we can discuss in more detail if y want

File: src/Nox.Localization.SqlServer/Nox.Localization.SqlServer.csproj
Comment (jan-schutte) [reply src/Nox.Localization.SqlServer/Nox.Localization.SqlServer.csproj:1]: Merged DB context into Nox entity store

**Comment by rochar** [comment]
close, until we get back to localization

**Pull request closed without merging**